### PR TITLE
Misc Tweaks

### PR DESCRIPTION
**Pull request by jcstringer**
* Force an updated_at timestamp when records haven't changed on a POST
* Scope syncs endpoints to a client

### CODE REVIEW

File: app/controllers/g5_updatable/syncs_controller.rb [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
   end
 
   private
 
+  def  max_updated_at
+    arel = G5Updatable::Location
+    arel = arel.where(urn: params[:urn]) if params[:urn]
+    arel.maximum(:updated_at)
+  end  
+
```
Comment (perryqh) [inline app/controllers/g5_updatable/syncs_controller.rb]: Should this be in the model?
```
G5Updatable::Location.max_updated_at(params[:urn])
```

**Comment by perryqh** [comment]
:+1:

File: app/controllers/g5_updatable/syncs_controller.rb (hunk 1 above)
Comment (jcstringer) [inline app/controllers/g5_updatable/syncs_controller.rb]: Good point

**Comment by perryqh** [comment]
:100:

**Pull request merged**